### PR TITLE
story/cord 11

### PR DESCRIPTION
**Pull request by aa-kashk**

### CODE REVIEW

**Comment by jdamerow** [comment]
Jenkins, test this please.

File: cord19-explorer/src/main/webapp/WEB-INF/views/paper.html [hunk 1]
```diff
@@ -67,6 +67,8 @@ <h3 th:text="*{section.title != ''} ? *{section.title}: 'No Section title'"></h3
 	      	[# th:each="match : ${para.locationMatches}" ]
 	      	[# th:if="${match != null && match.selectedArticle != null}" ]
 	      	$("#[(${#ids.next('para')})]").mark("[(${match.locationName})]", { "separateWordSearch": false})	
+	     	var title = [[${#strings.defaultString(publication.metadata.title, 'No title available')}]];
+	      	$("#[(${#ids.next('para')})]").mark("[(${match.locationName})]", { "separateWordSearch": false}).attr('title', title);
```
Comment (jdamerow) [inline cord19-explorer/src/main/webapp/WEB-INF/views/paper.html:71]: the title should be the title of the found match, not the publication title

**Review by jdamerow — CHANGES_REQUESTED** [review]

**Comment by aa-kashk** [comment]
not sure if this is this is correct as this shows the location instead of a title.

**Comment by jdamerow** [comment]
Jenkins, test this please.

**Comment by jdamerow** [comment]
Looks good I think. let's make that a link though that links to the wikipedia article in a new tab (for the url, basically just the wikipedia url prefix + the term should do it )

**Comment by aa-kashk** [comment]
(for the url, basically just the wikipedia url prefix + the term should do it ) can you elaborate? I am unable to figure out the wiki url prefix variable and the term that needs to be appended.  I have implemented anchor tag to open a new page on the matched paragraph.

**Comment by jdamerow** [comment]
Something is going wrong. It turns the whole paragraph into a link and the hover text is wrong (in the page in the screenshot, it says "Italy" for Wuhan).
image

**Pull request closed without merging**